scripts/start_registry_and_push_images.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Remove existing container named 'registry' if it exists
+if docker ps -a --format '{{.Names}}' | grep -Eq '^registry$'; then
```

Comment:
One other option is just uploading these prebuilt images to GHCR and getting rid of the build step altogether. It would make everyone's lives easier, probably.